webusb/idlharness.html

```diff
@@ -0,0 +1,216 @@
+        readonly attribute USBTransferStatus status;
+      };
+
+      [Constructor(USBTransferStatus status, optional DataView? data)]
+      interface USBIsochronousInTransferPacket {
+        readonly attribute DataView? data;
+        readonly attribute USBTransferStatus status;
+      };
+
+      [Constructor(sequence<USBIsochronousInTransferPacket> packets, optional DataView? data)]
+      interface USBIsochronousInTransferResult {
+        readonly attribute DataView? data;
+        readonly attribute FrozenArray<USBIsochronousInTransferPacket> packets;
+      };
+
+      [Constructor(USBTransferStatus status, optional unsigned long bytesWritten = 0)]
+      interface USBIsochronousOutTransferPacket {
+        readonly attribute unsigned long bytesWritten;
+        readonly attribute USBTransferStatus status;
+      };
+
+      [Constructor(sequence<USBIsochronousOutTransferPacket> packets)]
+      interface USBIsochronousOutTransferResult {
+        readonly attribute FrozenArray<USBIsochronousOutTransferPacket> packets;
+      };
+
+      [Constructor(USBDevice device, octet configurationValue)]
+      interface USBConfiguration {
+        readonly attribute octet configurationValue;
+        readonly attribute DOMString? configurationName;
+        readonly attribute FrozenArray<USBInterface> interfaces;
+      };
+
+      [Constructor(USBConfiguration configuration, octet interfaceNumber)]
+      interface USBInterface {
+        readonly attribute octet interfaceNumber;
+        readonly attribute USBAlternateInterface alternate;
+        readonly attribute FrozenArray<USBAlternateInterface> alternates;
+        readonly attribute boolean claimed;
+      };
+
+      [Constructor(USBInterface deviceInterface, octet alternateSetting)]
+      interface USBAlternateInterface {
+        readonly attribute octet alternateSetting;
+        readonly attribute octet interfaceClass;
+        readonly attribute octet interfaceSubclass;
+        readonly attribute octet interfaceProtocol;
+        readonly attribute DOMString? interfaceName;
+        readonly attribute FrozenArray<USBEndpoint> endpoints;
+      };
+
+      enum USBDirection {
+        "in",
+        "out"
+      };
+
+      enum USBEndpointType {
+        "bulk",
+        "interrupt",
+        "isochronous"
+      };
+
+      [Constructor(USBAlternateInterface alternate, octet endpointNumber, USBDirection direction)]
+      interface USBEndpoint {
+        readonly attribute octet endpointNumber;
+        readonly attribute USBDirection direction;
+        readonly attribute USBEndpointType type;
+        readonly attribute unsigned long packetSize;
+      };
+    </script>
+    <script>
+      'use strict';
+
+      var idl_array = new IdlArray();
+      idl_array.add_untested_idls(document.querySelector('#untested').textContent);
+      idl_array.add_idls(document.querySelector('#tested').textContent);
+      idl_array.add_objects({
+        Navigator: ['navigator'],
```

Comment:
I don't think we need this object, since `navigator` is untested and l.205 uses the context navigator, right?